src/Compression/LZ4_decompress_faster.cpp

```diff
@@ -561,7 +569,11 @@ bool NO_INLINE decompressImpl(
 
         /// Get match offset.
 
+#if defined(__BYTE_ORDER__) && (__BYTE_ORDER__ == __ORDER_BIG_ENDIAN__)
```

Comment:
Let's remove `defined(__BYTE_ORDER__)` because otherwise, it's error-prone.

Reply:
Fixed